homework4/exercise2.php

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Homework4</title>
+</head>
+<body>
+	<header>
+		<h2>Homework4</h2>
+	</header>
+	<main>
+	<h3>Exercise2</h3>
+	<?php
+	$n=18;
+	for ($j=17; $j < $n; $j++) { 
```

Comment:
@Niki1993 
Why do you set the $j to 17 here ?. 2 is a good start if you want to prime the first "n" primes.